tenji240/cbp-theme
# Tenji240/Font Awesome upgrade to v5.4.1

#### What's this PR do?
- This Adds Font Awesome 5.4.1 Support for the CBP Theme

#### Where should the reviewer start?
- I'd start by reviewing the package.json and build methods, double check the file paths, and other font-awesome custom code from version 4 that have been updated to match up with version 5

#### How should this be manually tested?
- Fire up Kitchen Sink and look out for any missing icons or such. Cross-browser check + IE 11 Check

#### Any background context you want to provide?
- In a project, we needed icons from the latest FA, that were not available in FA v4

#### What are the relevant tickets?
- none

#### Screenshots (if appropriate)
- none

#### Questions:
- Is there a blog post? Nope
- Does the knowledge base need an update? Yes. Please adhere to FA v5 [nomenclature](https://fontawesome.com/how-to-use/on-the-web/referencing-icons/basic-use) `.fas` prefix
